CONTRIBUTING.md

```diff
@@ -31,6 +44,51 @@ If you don't see your idea listed, do one of the following:
 ## Community
 Discussions about CircuitVerse issues and features take place on the repository's [Issues](https://github.com/CircuitVerse/CircuitVerse/issues) and [Pull Requests](https://github.com/CircuitVerse/CircuitVerse/pulls) sections. Anybody is welcome to join these conversations. See the [README](README.md) for more information on communication channels.
 
+## Code of Conduct
```

Comment:
We already have a coc file in the repo

Reply:
Is the rest of the stuff fine? I will remove this section if you want.